README.md

```diff
@@ -1,3 +1,19 @@
 # Tendermint Light Client
 
 PoC for a minimalistic TM Light Client based on [lib-rs](https://github.com/cosmos/ibc-rs)
+
+## Cli interface 
+
+### Fetch Consensus State and Header 
+This command fetch consensus state and header from full-node. This only use for testing `Verify` command.  
+
+### Verify 
+
+```bash
+tendermint-lightclient verify <CS_PATH> <HEADER_PATH> <NEW_CS_PATH>
```

Comment:
Could you describe the format of consensus state?
Is it verifying or updating the consensus? Based on the usage I guess it updates, but the section title says it is verifying

Reply:
I separate verifying and updating command also add format about Consensus State to README.